### PR TITLE
Updated documentation

### PR DESCRIPTION
**Pull request by jaswantcoder**
*Issue # updated documentation*

### CODE REVIEW

File: EXAMPLE.md [hunk 1]
```diff
@@ -6,31 +8,40 @@ import {
...
+### Creating KMS Client
 
+```ts
 const kmsClient: KMSClient = new KMSClient({
-  region: 'us-west-1',
+  region: 'REGION',
   credentials: {
     accessKeyId: 'EXAMPLEACCESSKEY',
     secretAccessKey: 'examplesecretkey/K',
   },
```
Comment (debanshuk) [inline EXAMPLE.md]: Nit: We can skip this part from the example, as we don't want to promote usage of IAM users.

File: EXAMPLE.md [hunk 2]
```diff
@@ -6,31 +8,40 @@ import {
...
 const signingOpts = {
   format: 'compact',
-  compact: true,
   fields: {
-    exp: 1714194877000,
-    alg: 'RSASSA_PSS_SHA_512',
-    kid: 'arn:aws:kms:eu-west-1:123456123456:alias/ExampleSigningKey',
-    crit: ['exp'],
+    alg: 'SIGNING_ALGORITHM',
+    kid: 'KMS_SIGNING_KEY_ID',
```
Comment (debanshuk) [inline EXAMPLE.md:30]: At some place in this file we should mention that 'KMS_SIGNING_KEY_ID' (and other KMS key IDs) can be key-id, key-id-arn, key-alias or key-alias-arn, based upon the user's needs.

File: EXAMPLE.md [hunk 3]
```diff
@@ -43,22 +54,31 @@ const jws: Promise<string> = signer
     return result;
   });
 
-// Encryption
+  ```
+
+### Encryption
+
+1. Prepare encryption options with format as `compact`, header fields and cek as KMSSymmetricCEK object.
+2. Prepare encryption recipient with reference as `false`, and key as KMSSymmetricKey object. Use same kms key to create KMSSymmetricCEK and KMSSymmetricKey.
```
Comment (debanshuk) [inline EXAMPLE.md]: Nit: 'kms' -> 'KMS'.

**Comment by mjsheikh** [comment]
Please enable atleast the following branch protection rules for the `main` branch with the following settings:
1. Require a pull request before merging
a. Require approvals
b. Dismiss stale pull request approvals when new commits are pushed

File: EXAMPLE.md (hunk 1 above)
Comment (jaswantcoder) [reply EXAMPLE.md]: removed

File: EXAMPLE.md (hunk 2 above)
Comment (jaswantcoder) [reply EXAMPLE.md:30]: added a Note at the end of file

File: EXAMPLE.md (hunk 3 above)
Comment (jaswantcoder) [reply EXAMPLE.md]: done

**Review by mjsheikh — APPROVED** [review]

**Review by debanshuk — COMMENTED** [review]
It seems there a 4 commits in this PR with same/similar commit messages. Could you please merge these commits to a single commit?

**Comment by jaswantcoder** [comment]
> It seems there a 4 commits in this PR with same/similar commit messages. Could you please merge these commits to a single commit?

Yes, squashed commits into one and merged. Deleted branch

**Pull request merged**